tests/Jenkins/dirac_ci.sh

```diff
@@ -289,6 +289,55 @@ function fullInstallDIRAC(){
 }
 
 
+#...............................................................................
```

Comment:
This was already backported by @fstagni so @atsareg has to be careful not to propagate Fderico's PR to further branches.

Reply:
yes, just remeber to rebase after @atsareg merge the various PR (and rel-v6r20 in integration)